build.gradle

```diff
@@ -21,3 +21,8 @@ eclipseResourceFilters {
 	exclude().folders().name('plugin-maven')
 	exclude().folders().name('testlib')
 }
+
+task wrapper(type: Wrapper) {
+	gradleVersion = '4.4.1'
+	distributionUrl = "https://services.gradle.org/distributions/gradle-$gradleVersion-all.zip"
+}
```

Comment:
Please remove this task.  You can do this without adding a task to build.gradle:

```
gradle wrapper --gradle-version 4.4.1
```